e2e/test-runner/runner_test.go

```diff
@@ -33,3 +58,49 @@ func TestBasicSetup(t *testing.T) {
 
 	// TODO: Assert more. We have other expectations about the most basic setup.
 }
+
+// This tests the experience that most users have:
+// opening WSL from the store or from the Start menu
+func TestDefaultExperience(t *testing.T) {
+	tester := WslTester(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	shellCommand := []string{"-noninteractive", "-nologo", "-noprofile", "-command",
+		*launcherName, "--hide-console"}
+	cmd := exec.CommandContext(ctx, "powershell.exe", shellCommand...)
+
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &out
+
+	cmd.Start()
+	time.Sleep(1 * time.Second)
+
+	// Polling until status is no longer Installing
+	status := getDistroStatus(&tester)
+	for status == "Installing" {

```

Comment:
This Sleep(1 second) is a bit ugly but I couldn't think of a better option to avoid failing the loop condition. Suggestions are welcome.

Reply:
Solved!